crossbar/router/observation.py

```diff
@@ -43,8 +43,8 @@ def is_protected_uri(uri):
 
 class OrderedSet(set):
 
-    def __init__(self, iter=None):
-        super(set, self).__init__(iter)
+    def __init__(self):
```

Comment:
set() on Python 3 works differently, and we don't instantiate it with any arguments in the codebase (it also doesn't put iter in `self._list`).